sample_factory_examples/mujoco_examples/README.md

```diff
@@ -0,0 +1,21 @@
+## Installation
+
+### Method 1 - Install from setup.py on sf2
+
+1. Clone sample factory and checkout the sf2 branch and `cd sample-factory`
+2. Create new conda environment with python=3.9 `conda create -n {myenv} python=3.9`
+3. Install pytorch with `conda install pytorch==1.11.0 torchvision -c pytorch`
+4. Install other dependencies with `pip install -e .`
+5. Install gym environments `pip install "gym[mujoco]"`
+6. `pip install mujoco`
+7. `pip install mujoco-py`
+
+### Method 2 - Install from environment.yaml from master (also steps for debugging on existing environment)
```

Comment:
We decided not to support the yml configuration file and just focus on setup.py dependencies (otherwise there are two installation procedures we need to support).
So we're probably good with just one installation method.